Dockerfile

```diff
@@ -52,19 +63,27 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
         ruby-bundler \
         ruby-dev \
         zlib1g-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
     ;
 
 WORKDIR /build
 
-COPY . .
-RUN rake build
-RUN vagrant plugin install ./pkg/vagrant-libvirt*.gem
+# comma-separated list of other supporting plugins to install
+ARG DEFAULT_OTHER_PLUGINS=vagrant-mutate
 
-RUN for dir in boxes data tmp; \
-    do \
-        touch /vagrant/${dir}/.remove; \
-    done \
-    ;
+COPY . .
+RUN rake build \
+    && vagrant plugin install ./pkg/vagrant-libvirt*.gem \
+    && for plugin in $(echo "$DEFAULT_OTHER_PLUGINS" | sed "s/,/ /g"); \
+         do \
+           vagrant plugin install ${plugin} ; \
+         done \
```

Comment:
This should probably be combined into a single call as otherwise I recall it repeats performing the install of all existing plugins along with each new plugin that is installed.